fdbclient/vexillographer/fdb.options

```diff
@@ -145,6 +145,9 @@ description is not currently required but encouraged.
     <Option name="transaction_max_retry_delay" code="502"
             paramType="Int" paramDescription="value in milliseconds of maximum delay"
             description="Set the maximum amount of backoff delay incurred in the call to ``onError`` if the error is retryable. This sets the ``max_retry_delay`` option of each transaction created by this database. See the transaction option description for more information." />
+    <Option name="transaction_size_limit" code="503"
+			paramType="Int" paramDescription="value in bytes"
+			description="Set the maximum transaction size which, if exceeded, will cause the transaction to be cancelled. Default to 10,000,000 bytes." />
```

Comment:
White space (`paramType` here should be aligned with `paramType` in `transaction_max_retry_delay`).